config/application.rb

```diff
@@ -44,10 +46,12 @@ class Application < Rails::Application
     # Currently 90 minutes is based on most 10G uploads on slow connections taking just under 1.5 hours
     config.active_storage.service_urls_expire_in = 90.minutes
 
-    console do
-      Honeybadger.configure do |config|
-        config.report_data = false
-      end
-    end
```

Comment:
Do we not need [this](https://github.com/sul-dlss/happy-heron/pull/1767/files) anymore?

Reply:
Eh, I'll restore.